web_scraper/load.py

```diff
@@ -21,11 +21,14 @@ def sanitise_filename(filename: str) -> str:
 
 def extract_title(url: str) -> str:
     """Extracts title used for s3_filename from given url."""
-    
-    page = urlopen(url)
-    soup = BeautifulSoup(page, 'html.parser')
-    title = soup.title.text.strip()
 
+    try:
+        page = urlopen(url)
+        soup = BeautifulSoup(page, 'html.parser')
+        title = soup.title.text.strip()
+    except ex:
+        return None
```

Comment:
This function is returning a string or none, do you want to change the type hint to show both? or edit it so it returns a string either way